drivers/bridge/port_mapping.go

```diff
@@ -13,7 +13,7 @@ import (
 
 var (
 	defaultBindingIP   = net.IPv4(0, 0, 0, 0)
-	defaultBindingIPV6 = net.ParseIP("::")
+	defaultBindingIPV6 = net.ParseIP("::1")
```

Comment:
I think this change does the opposite of what the commit description says, changing the default from unspecified to localhost/loopback and not the other way around.